### ISSUE TITLE
mkdocs

### ISSUE THREAD

**Issue by berot3** [sander76/markdown-columns#1]
Hi!
question: Can I use this in mkdocs? I am looking for an extension that provides columns in mkdocs...

**Comment by sander76** [comment]
Yes you can. I am using it myself. I must admit I am not sure whether it all works as I described in the readme. But it probably mostly will. So you'll need to do a bit of experimentation. PR's are more than welcome !

Sander.

**Comment by berot3** [comment]
Thanks for the reply, but how do you mean? Do you use columns in the form of HTML-tags?

**Comment by berot3** [comment]
Right, thanks. But how did you add this?

I am not sure where to put the extension inside of mkdocs

```
from md_columns import CssColumnsExtension

markdown.Markdown(extensions=[CssColumnsExtension()])
```

Usually mkdocs-extension are added to the `mkdocs.yml`.

**Comment by sander76** [comment]
@berot3 I just updated the readme. Have a look there. Please let me know if it works.
